apps/server/src/shared/infra/oauth-provider/oauth-provider.service.ts

```diff
@@ -1,5 +1,5 @@
 import { OauthClient } from '@shared/infra/oauth-provider/dto/interface/oauth-client.interface';
```

Comment:
Will there be multiple services(providers) extending this abstract class in the future?
If so, then I believe the methods with exceptions should be moved to the abstract class.
`https://github.com/hpi-schul-cloud/schulcloud-server/pull/3643/files#diff-8a01cb9f1ee04899d4d0c63f1b31388d4d3185d5703855f9b0c055654a0198cbR28-R58`

Reply:
This is a good idea. I implemented that